profiles/serializers.py

```diff
@@ -0,0 +1,114 @@
+from rest_framework import serializers
+from urllib.parse import urlparse
+from profiles.models import Startup, Investor
+from projects.serializers import ProjectSerializer
+from common.enums import Stage  # Updated import to reflect actual location of Stage enum
+from common.validation import SocialLinksValidationMixin  # Shared mixin for social_links validation
+
+
+class StartupShortSerializer(serializers.ModelSerializer):
+    """
+    Lightweight serializer for nested use (e.g., inside Investor).
+    """
+    class Meta:
+        model = Startup
+        fields = [
+            'id', 'company_name', 'industry', 'country',
+            'website', 'stage', 'is_participant'
+        ]
+        read_only_fields = fields
+
+
+class StartupSerializer(SocialLinksValidationMixin, serializers.ModelSerializer):
+    """
+    Full serializer for Startup.
+    Uses shared mixin for social_links validation.
+    Includes nested project details.
+    """
+    projects = ProjectSerializer(many=True, read_only=True)
+
+    class Meta:
+        model = Startup
+        fields = [
+            'id', 'company_name', 'description', 'industry',
+            'country', 'website', 'email', 'phone',
+            'contact_person', 'location', 'status',
+            'stage', 'social_links', 'is_participant',
+            'founded_at', 'team_size', 'is_active',
+            'user', 'created_at', 'updated_at',
+            'projects'
+        ]
+        read_only_fields = ['id', 'created_at', 'updated_at', 'projects']
```

Comment:
Should 'social_links' be required? If not, set 'required=False' in the field declaration.